### PR TITLE
SUIT-9587 Add hint for command-specific help messages to JS API

### PR DESCRIPTION
**Pull request by AntonSBilan**
SUIT-9587
Add hint for command-specific help messages to JS API

### CODE REVIEW

File: lib/testLauncher/buildArgs.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ const buildYargs = () => {
 		.boolean('help')
 		.boolean('version')
 		.help(false)
+		.epilog(`Run ${stColors.bold('suitest interactive --help')} or ${stColors.bold('suitest automated --help')} to see those command's options`)
```
[the remark this answers is not in the document]
Comment (andriichumak) [reply lib/testLauncher/buildArgs.js]: It's OK. Are you sure it's not added when user runs other commands then --help?

**Comment by andriichumak** [comment]
@AntonSBilan we found another issue that is somewhat related, would be great if you could fix it in scope of this one.
When user runs just `suitest` without options or command - it would output same as with `--help` argument. Note that `suitest -v` should still only output version, not help message.

File: lib/testLauncher/buildArgs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AntonSBilan) [reply lib/testLauncher/buildArgs.js]: For me it only appears with suitest --help

File: lib/testLauncher/buildArgs.js [hunk 2]
```diff
@@ -12,6 +14,8 @@ const buildYargs = () => {
 		.boolean('help')
 		.boolean('version')
 		.help(false)
+		.check((hash, argv) => !argv.length && stColors.errorColor(texts['tl.noArguments']()), false)
+		.epilog(`Run ${stColors.bold('suitest interactive --help')} or ${stColors.bold('suitest automated --help')} to see those command's options`)
```
Comment (andriichumak) [inline lib/testLauncher/buildArgs.js]: Could you move the text to texts? There is a way to inject variables with tagged template.

Comment (andriichumak) [reply lib/testLauncher/buildArgs.js]: Also, I think there is no need for apostrophe in "command's" after all, as it's plural.

File: lib/texts.js [hunk 3]
```diff
@@ -121,6 +121,7 @@ ${leaves}`,
 
 	// test launcher
 	'tl.nodevices': () => 'there is no device in config to proceed with testing',
+	'tl.noArguments': () => 'There was no argument or command specified',
```
Comment (andriichumak) [inline lib/texts.js]: Command was not specified

**Pull request merged**